e2e/specs/register-new-patient.spec.ts

```diff
@@ -40,9 +49,13 @@ test('Register a new patient', async ({ page }) => {
   });
 
   await test.step("And I should be redirected to the new patient's chart page", async () => {
-    const patientChartUrlRegex = new RegExp('^[\\w\\d:\\/.-]+\\/patient\\/[\\w\\d-]+\\/chart\\/.*$');
-    await page.waitForURL(patientChartUrlRegex);
-    await expect(page).toHaveURL(patientChartUrlRegex);
+    await page.waitForURL(PATIENT_CHART_URL);
+    await expect(page).toHaveURL(PATIENT_CHART_URL);
+
+    const url = page.url();
+    const match = url.match(PATIENT_CHART_URL);
+    patientUuid = match?.[1];
+    expect(patientUuid).toBeTruthy();
```

Comment:
Ditto this. We kind of already know that the value is truthy if the URL matched, no